test/import/FSHImporter.Extension.test.ts

```diff
@@ -78,17 +94,89 @@ describe('FSHImporter', () => {
         expect(extension.id).toBe('789');
       });
 
+      it('should parse an extension with multiple contexts', () => {
+        const input = `
+        Extension: SomeExtension
+        Parent: ParentExtension
+        Id: some-extension
+        Context: "some.fhirpath()", Observation.component, http://example.org/MyPatient#identifier,
+                 "another.fhirpath(var, 0)"
+        `;
```

Comment:
We need to add an assertion that confirms that no errors are emitted. As of now, this test _does_ produce errors because the parser doesn't know what to do w/ the spaces between the contexts.